gradle.properties

```diff
@@ -0,0 +1,14 @@
+useLocal=false
+pluginSinceBuild=241.14494.240
```

Comment:
why did this value need to change from `241`?

Reply:
yes, I noticed it should be 241. I will update it.